src/python/ensembl/io/genomio/seq_region/gbff.py

```diff
@@ -0,0 +1,104 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A SeqRecord wrapper."""
+
+__all__ = [
+    "GBFFRecord",
+]
+
+from dataclasses import dataclass
+import re
+
+from Bio.SeqRecord import SeqRecord
+
+from ensembl.io.genomio.seq_region.exceptions import UnknownMetadata
+
+##############################################
```

Comment:
I'm not too fond of these lines as they usually clutter IDEs more than an empty line, but won't complain.